docker-images/telegraf/Dockerfile

```diff
@@ -1,6 +1,6 @@
 # Telegraf agent configured for Wavefront output intended to be used as it's own deployment to collect metrics
 
-FROM telegraf:1.21.2
+FROM telegraf:1.22.0-alpine
```

Comment:
@skajagar Why alpine here?

Reply:
Saw the request for base image upgrade here - https://github.com/wavefrontHQ/wavefront-kubernetes/pull/18
also v1.22.0 alpine is available so updating it